src/test/java/com/trycloud/utility/BrowserUtil.java

```diff
@@ -48,5 +48,14 @@ public static void waitForElementVisibility(WebElement element){
 
         explicitWait.until(ExpectedConditions.visibilityOf(element));
     }
+    /**
```

Comment:
Such inspirational method that I have never seen.

![image](https://github.com/user-attachments/assets/51fc370b-203c-4b9e-9f20-cf2cd81f2c65)